### DISCUSSION TITLE
Stop Git Integration

### DISCUSSION THREAD

**Discussion by rtrad89** [apache/netbeans#4505] [Q&A] [answered]
Hi all
I use Netbeans 12.6, and I was wondering how to disable all Git integration because we want to standardise on Git CLI. How to disable Git operations in Netbeans?

I tried to look for a plugin to disable, to no avail.
Thanks!

**Comment by neilcsmith-net** [comment, marked as answer]
Go to `Tools / Plugins / Installed`.  Make sure `Show details` is selected.  Use search to find `Git` module. Select and deactivate.

> I use Netbeans 12.6

Current is NetBeans 14, and soon to be NetBeans 15.
